packages/venia-ui/lib/components/MegaMenu/megaMenuItem.css

```
@@ -18,6 +18,15 @@
     display: flex;
 }
 
+.megaMenuItem_active {
+    composes: megaMenuItem;
+    display: flex;
+}
+
+.arrowDown {
+    padding-left: 8px;
```

Prefer rem units here
```suggestion
    padding-left: 0.5rem;
```